src/EnergyPlus/HVACControllers.cc

```diff
@@ -1319,6 +1322,31 @@ namespace HVACControllers {
                     // (i.e., reset MinAvailActuated based on Node%MassFlowRateMaxAvail)
                     ControllerProps(ControlNum).MinAvailActuated =
                         min(ControllerProps(ControlNum).MinAvailActuated, ControllerProps(ControlNum).MaxAvailActuated);
+
+                    // If the Node flow rate is zero (or just < MinAvail?), we override the min value to zero
+                    if (ControllerProps(ControlNum).ActuatedValue < ControllerProps(ControlNum).MinAvailActuated) {
+
+                        if (!WarmupFlag) {
+                            if (ControllerProps(ControlNum).MinFlowOverrideErrorIndex == 0) {
+                                ShowWarningError("Controller:WaterCoil '" + ControllerProps(ControlNum).ControllerName
+                                               + "', Minimum control flow is greater than requested flow rate:");
+                                ShowContinueError(" Minimum avail actuated=" + General::TrimSigDigits(ControllerProps(ControlNum).MinAvailActuated, NumSigDigits));
+                                ShowContinueError(" Node Mass Flow Rate=" + General::TrimSigDigits(ControllerProps(ControlNum).ActuatedValue, NumSigDigits));
+                                ShowContinueError(" Resetting Mininimum Available Actuated to Node Flow. Check Inputs in Controller:WaterCoil.");
+                                ShowContinueErrorTimeStamp("");
+                            }
+                            ShowRecurringWarningErrorAtEnd("Controller:WaterCoil '" + ControllerProps(ControlNum).ControllerName
+                                    + "', Minimum control flow is greater than requested flow rate hand has been reset.",
+                                    ControllerProps(ControlNum).MinFlowOverrideErrorIndex,
+                                    ControllerProps(ControlNum).MinAvailActuated, // Report Max
+                                    ControllerProps(ControlNum).MinAvailActuated, // Report Min
+                                    _,             // Don't report Sum
+                                    "{kg/s}",         // Max Unit
+                                    "{kg/s}");        // Min Unit
+                        }
+
+                        ControllerProps(ControlNum).MinAvailActuated = ControllerProps(ControlNum).ActuatedValue;
```

Comment:
While looking at this I was thinking that this might be a good way to control flow with min flow > 0. If min flow is lower than min avail, then turn the coil off. It will turn on later when the load has increased. I'm not sure how that is handled in the real world. Then I get to this line and see that MinAvailActuated in reset. This is of course different than the user input for MinVolFlowActuated but is still a control point. Should MinAvailActuated ever be below the user input for MinVolFlowActuated ? unless of course it is 0.

Shouldn't this be:

    ControllerProps(ControlNum).ActuatedValue = ControllerProps(ControlNum).MinAvailActuated;

or better yet shut down flow so the coil is off?

    ControllerProps(ControlNum).ActuatedValue = 0.0;